Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# More Parameters/ options (Time/Punishments/Rally points/AI Recruitment)

## What type of PR is this?
1. [ ] Bug
2. [x] Change
3. [x] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:
**Added:**
Parameters
- Disable rally points
- Disable player recruiting units
- Disable invader punishments
- more options for time multiplier 1:2 1:3

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
check if there is a way to recruit units to player squad with parameter set to disabled
********************************************************
Notes:
